### PR TITLE
Rewrite "composite approaches" section

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose

Addresses #128. This attempts to tie how the challenges presented throughout the manuscript are addressed by the approaches highlighted in this section (DeepProfile and MultiPLIER).

### Directions for reviewers

I'll note that, ideally, before this goes in someone takes another pass (or we plan another pass prior to submission), as I didn't closely read this after writing it and spending time away like I like to. But to address the first bulletpoint below, someone besides me needs to look!

- This may not be exactly what @jaybee84 intended with #128, but does it address the _spirit_ of the issue?
- I certainly didn't make the section shorter. What can I cut?
- I don't think the versioned DOI for DeepProfile is going to work - we might need to use the URL (versioning _does_ work for that according to the [FAQ](https://www.biorxiv.org/about/FAQ)). Has anyone tried to tie something to a specific bioRxiv version elsewhere in the manuscript?
- VAEs are included in the definition section and specifically text that used to live in this section. How do we link to that definition?

#### Is the pull request ready for review?

Yes.

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/06.multiple-approaches-required.md [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
-The authors used the learned attributes to encode a low-dimensional representation of held-out AML data with drug response labels, and used this representation as input to a supervised classifier that predicted in vitro drug response, which outperformed using the original features. 
-The study by Dincer and colleagues highlights another challenge: samples collected as part of multiple studies may not be associated with the deep phenotypic information that would maximize their scientific value, but this does not preclude the use of these samples in learning representations.
+Thousands of acute myeloid leukemia (AML) patient gene expression samples have been collected over time and are publicly available.
+However, as is often the case with publicly available data, not all of these samples include the most relevant clinical or phenotypic data such as drug response.
+The authors of Dincer et al. had access to an _in vitro_ experiment that examined the response to 160 drugs for 30 AML patient samples, measured using genome-wide arrays which have tens of thousands of features [@doi:10.1038/s41467-017-02465-5].
```
Comment (jaybee84) [inline content/06.multiple-approaches-required.md]:
```suggestion
```

File: content/06.multiple-approaches-required.md [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+Training on rare disease datasets with a low number of samples, a considerable number of tasks, and many features can result in ML models that are of limited utility.
+The low number of samples in the drug response dataset could also be prohibitive when performing representation learning.
+Instead, the authors trained a variational autoencoder on over 6500 AML samples (VAE; see [definitions]) [TODO: link between sections?] to reduce the dimensionality of the test set in an approach termed DeepProfile [@doi:10.1101/278739v2].
```
Comment (jaybee84) [inline content/06.multiple-approaches-required.md]:
```suggestion
In view of these challenges, an _in vitro_ experiment that examined the response to 160 drugs for 30 AML patient samples, measured using genome-wide arrays that have tens of thousands of features was capitalized upon [@doi:10.1038/s41467-017-02465-5].
The authors trained a variational autoencoder on over 6500 AML samples (VAE; see [definitions]) [TODO: link between sections?] to reduce the dimensionality of the test set in an approach termed DeepProfile [@doi:10.1101/278739v2].
```

File: content/06.multiple-approaches-required.md [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
+While DeepProfile was centered on training on an individual disease and tissue combination, some rare diseases affect multiple tissues that a researcher may be interested in studying together for the purpose of biological discovery. 
+Studying multiple tissues poses challenges – features that can be measured an appreciable level may be tissue-specific even when looking at learned features or representations, cross-tissue analysis may require an analyst to compare representations from multiple models and models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.
+To address these challenges, Taroni et al. trained Pathway-Level Information ExtractoR (PLIER) [@doi:10.1038/s41592-019-0456-1] on a large generic collection of human transcriptomic data (recount2 [@doi:10.1038/nbt.3838]) and used the latent variables learned by the model to describe transcriptomic data from the unseen rare diseases antineutrophil cytoplasmic antibody (ANCA)-associated vasculitis (AAV) and medulloblastoma in an approach termed MultiPLIER [@doi:10.1016/j.cels.2019.04.003].
```
Comment (jaybee84) [inline content/06.multiple-approaches-required.md]:
```suggestion
To address these challenges, a Pathway-Level Information ExtractoR (PLIER) [@doi:10.1038/s41592-019-0456-1] was trained on a large generic collection of human transcriptomic data (recount2 [@doi:10.1038/nbt.3838]).
The latent variables learned by the model were used to describe the transcriptomic data from unseen rare diseases antineutrophil cytoplasmic antibody (ANCA)-associated vasculitis (AAV) and medulloblastoma in an approach termed MultiPLIER [@doi:10.1016/j.cels.2019.04.003]. 
```

File: content/06.multiple-approaches-required.md [hunk 4]
```diff
@@ -1,23 +1,33 @@
...
-PLIER is a matrix factorization approach that takes prior knowledge in the form of gene sets or pathways and gene expression data as input. 
+While DeepProfile was centered on training on an individual disease and tissue combination, some rare diseases affect multiple tissues that a researcher may be interested in studying together for the purpose of biological discovery. 
+Studying multiple tissues poses challenges – features that can be measured an appreciable level may be tissue-specific even when looking at learned features or representations, cross-tissue analysis may require an analyst to compare representations from multiple models and models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.
```
Comment (jaybee84) [inline content/06.multiple-approaches-required.md]:
```suggestion
Studying multiple tissues poses significant challenges – features that can be appreciably measured may be tissue-specific even when looking at learned features or representations, a cross-tissue analysis may require an analyst to compare representations from multiple models and models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.
```

**Review by jaybee84 — APPROVED** [review]
I added a few minor edits.

We can discuss the TODOs in today's meeting :)

**Comment by jaclyn-taroni** [comment]
I will file an issue for linking between sections and then use the URL to tie to a specific version of DeepProfile.

File: content/06.multiple-approaches-required.md [hunk 5]
```diff
@@ -1,23 +1,33 @@
...
-The study by Dincer and colleagues highlights another challenge: samples collected as part of multiple studies may not be associated with the deep phenotypic information that would maximize their scientific value, but this does not preclude the use of these samples in learning representations.
+Thousands of acute myeloid leukemia (AML) patient gene expression samples have been collected over time and are publicly available.
+However, as is often the case with publicly available data, not all of these samples include the most relevant clinical or phenotypic data such as drug response.
+The authors of Dincer et al. had access to an _in vitro_ experiment that examined the response to 160 drugs for 30 AML patient samples, measured using genome-wide arrays which have tens of thousands of features [@doi:10.1038/s41467-017-02465-5].
+Training on rare disease datasets with a low number of samples, a considerable number of tasks, and many features can result in ML models that are of limited utility.
+The low number of samples in the drug response dataset could also be prohibitive when performing representation learning.
```
Comment (jaclyn-taroni) [inline content/06.multiple-approaches-required.md]: If taking the above suggestion, this sentence would also need to be moved later in my opinion.

**Comment by jaclyn-taroni** [comment]
@jaybee84 I know you approved, but I'd like you to take another look before this goes in! I tried to address your comments without changing sentences to be in passive voice where possible.

**Review by jaybee84 — APPROVED** [review]
My earlier edits mostly removed the author names from the text, but I see that you have kept them in to avoid using passive voice. It's just a matter of personal preference (for me) to not have author names in the text. I am okay with your decision to keep them in for better readability.

**Pull request merged**